convention-develocity-gradle-plugin/plugins/gradle-5-or-newer/src/main/java/com/myorg/ConventionGradleEnterpriseGradlePlugin.java

```diff
@@ -14,7 +14,7 @@
 import org.gradle.util.GradleVersion;
 
 /**
- * An example Gradle plugin for enabling and configuring Gradle Enterprise features for
+ * An example Gradle plugin for enabling and configuring Develocity features for
  * Gradle versions 5.x and higher.
  */
 public class ConventionGradleEnterpriseGradlePlugin implements Plugin<Object> {
```

Comment:
Shouldn't we rename the class and method name?

Reply:
I'll rename the class but leave the renaming of the method name for when the GradleEnterprise extension is renamed.